src/class-convertkit-resource.php

```diff
@@ -484,6 +484,19 @@ public function get_cron_event() {
 
 	}
 
+	/**
+	 * Returns the timestamp for when the WordPress Cron event is next scheduled to run.
+	 *
+	 * @since   1.3.8
+	 *
+	 * @return  bool|int
+	 */
+	public function get_cron_event_next_scheduled() {
```

Comment:
This seems random, didn't know if it was a mistake to include on this PR.

Reply:
This is used by the upcoming Broadcasts to Posts functionality, to display when the cron event will next run:
https://github.com/ConvertKit/convertkit-wordpress/blob/broadcasts-to-posts-change-api-endpoint/admin/section/class-convertkit-admin-settings-broadcasts.php#L196

Apologies for not including that in the PR description.